doc/OneStepSetup.md

```diff
@@ -14,8 +14,8 @@ This is a streamlined process for setting up the Pi. You'll flash a preconfigure
 
     In Raspberry Pi Imager, you need to click 'Operating System' and then scroll _all the way down_ and select the 'Use custom' option.
 
-1.  Mount the card again, and in the `boot` directory create a `teslausb_setup_variables.conf` file to export the same environment variables normally needed for manual setup (including archive info, Wifi, and push notifications (if desired).
-    A sample conf file is located in the `boot` folder on the SD card. The latest sample is also available [from GitHub](https://github.com/marcone/teslausb/blob/main-dev/pi-gen-sources/00-teslausb-tweaks/files/teslausb_setup_variables.conf.sample).
+1.  Mount the 'boot'-labeled partition on the card again, and create a `teslausb_setup_variables.conf` file to export the same environment variables normally needed for manual setup (including archive info, Wifi, and push notifications (if desired).
```

Comment:
While technically correct, I think this is overly complicating things for people that aren't as tech savvy. For most people, simply inserting the sd card again will make it automatically mount the relevant partition again, with no need to specifically seek out and mount the partition.
I'd be OK with changing "boot directory" in the original sentence to "the drive labeled boot" or something similar. (using "drive" instead of "partition" because on Windows, MacOS and probably most Linux desktops this would just show up as a drive)